.github/workflows/bbw_build_container.yml

```diff
@@ -37,6 +43,11 @@ jobs:
             tag: debian12
             branch: 10.11
             platforms: linux/amd64, linux/arm64/v8, linux/ppc64le
+          - dockerfile: debian.Dockerfile
```

Comment:
Why? We don't have a 32 bit hardware to test. Its unclear if tests/builds/packages actually represent something that works on 32 bit hardware. So who's the real user?

Reply:
Same answer as above - we have and test in the old buildbot, seems to be a requirement for moving to the new one